doc/connecting-to-hosted-databases.md

```diff
@@ -0,0 +1,56 @@
+# Connecting to hosted databases
```

Comment:
Recommend using the "rich diff" view of this in the top right if you've not seen it before